subsys_tello_sensors.py

```diff
@@ -27,6 +20,31 @@ class TelloSensors:
                BackgroundFrameRead] = None
     mode: int = -1
 
+    # Dictionary containing every Tello internal variables that are retrieved when the
+    # Tello.get_current_state() method is called
+    tello_internal_state_variables: dict = {'pitch': 0,
```

Comment:
I dont Really like the storing mechanics : this way you have to get the whole dictionary each time you need a variable out of it.
Plus to accommodate that you're creating another disctionnary with duplicate variables which can leads to potential problems.

I Would recommend storing each variable as a class attribute only accessible in read access (i'm not sure if it's doable in python, see public private and protected variables in other languages).

Then if you want to use a dictionary for display purpose, create it with the _get_display_dict() method with the variable you need